src/main/java/com/osiris/autoplug/client/Main.java

```diff
@@ -266,6 +292,9 @@ public static void main(String[] _args) {
             } catch (Exception e) {
                 AL.warn(e);
             }
+            
+            SSHManager sshManager = SSHManager.getInstance(sshConfig);
+            sshManager.start();
```

Comment:
It seems like there is only one instance of this class anyways, and only one ssh server anyways, so you can make it a fully static class

Reply:
done (assuming you meant for me to convert the SSHServerSetup class to a fully static class)
idk if thats what u meant, im ✨*special*✨

Reply:
I actually meant the Ssh manager class

Reply:
done